docs/references/javascript/sign-up/sign-up.mdx

```diff
@@ -68,7 +68,7 @@ function create(params: SignUpCreateParams): Promise<SignUpResource>;
 | `web3Wallet` | `string \| null` | The user's Web3 wallet public address. In [Ethereum](https://docs.metamask.io/guide/common-terms.html#address-public-key), the address is made up of `0x` + `40 hexadecimal characters`. |
 | `username` | `string \| null` | The user's username. This option is available only if usernames are enabled. Please check the instance settings for more information. |
 | `unsafeMetadata` | `{[string]: any} \| null` | Metadata that can be read and set from the frontend. Once the sign-up is complete, the value of this field will be automatically copied to the newly created user's unsafe metadata. One common use case for this attribute is to use it to implement custom fields that can be collected during sign-up and will automatically be attached to the created `User` object. |
-| `strategy` | `'oauth_<provider>' \| 'saml' \| 'ticket'` | The strategy to use for the sign-up flow.<br />The following strategies are supported:<br /><ul><li>`oauth_<provider>`: The user will be authenticated with their social sign-in account. See available [OAuth providers](/docs/references/javascript/types/oauth#o-auth-provider).</li><li>`saml`: The user will be authenticated with SAML.</li><li>`ticket`: The user will be authenticated via the ticket *or token* generated from the Backend API.</li></ul> |
+| `strategy` | `'oauth_<provider>' \| 'saml' \| 'ticket'` | The strategy to use for the sign-up flow.<br />The following strategies are supported:<br /><ul><li>`oauth_<provider>`: The user will be authenticated with their social sign-in account. See available [OAuth providers](/docs/references/javascript/types/oauth#o-auth-provider).</li><li>`saml`: The user will be authenticated with SAML.</li><li>`ticket`: The user will be authenticated via the ticket *or token* generated from the Backend API.</li><li>`google_one_tap`: The user will be authenticated with Google One Tap UI. It's recommended to use [`authenticateWithGoogleOneTap()`](/docs/components/authentication/google-one-tap#authenticate-with-google-one-tap) instead, as it will also set the user's current session as active.</li></ul> |

```

Comment:
```suggestion
| `strategy` | `'oauth_<provider>' \| 'saml' \| 'ticket'` | The strategy to use for the sign-up flow.<br />The following strategies are supported:<br /><ul><li>`oauth_<provider>`: The user will be authenticated with their social sign-in account. See available [OAuth providers](/docs/references/javascript/types/oauth#o-auth-provider).</li><li>`saml`: The user will be authenticated with SAML.</li><li>`ticket`: The user will be authenticated via the ticket *or token* generated from the Backend API.</li><li>`google_one_tap`: The user will be authenticated with the Google One Tap UI. It's recommended to use [`authenticateWithGoogleOneTap()`](/docs/components/authentication/google-one-tap#authenticate-with-google-one-tap) instead, as it will also set the user's current session as active for you.</li></ul> |

```